skafka/src/main/scala/com/evolutiongaming/skafka/consumer/ConsumerMetrics.scala

```diff
@@ -115,50 +118,226 @@ object ConsumerMetrics {
         labels    = LabelNames("client", "topic")
       )
     } yield { (clientId: ClientId) =>
-      new ConsumerMetrics[F] {
+      new Summaries(
+        callsCounter,
+        resultCounter,
+        latencySummary,
+        recordsSummary,
+        bytesSummary,
+        rebalancesCounter,
+        topicsLatency,
+        ageSummary,
+        clientId
+      )
+    }
+  }
 
-        def call(name: String, topic: Topic, latency: FiniteDuration, success: Boolean) = {
-          val result = if (success) "success" else "failure"
-          for {
-            _ <- latencySummary.labels(clientId, topic, name).observe(latency.toNanos.nanosToSeconds)
-            _ <- resultCounter.labels(clientId, topic, name, result).inc()
-          } yield {}
-        }
+  private final class Summaries[F[_]: Monad](
+    callsCounter: LabelValues.`3`[Counter[F]],
+    resultCounter: LabelValues.`4`[Counter[F]],
+    latencySummary: LabelValues.`3`[Summary[F]],
+    recordsSummary: LabelValues.`2`[Summary[F]],
+    bytesSummary: LabelValues.`2`[Summary[F]],
+    rebalancesCounter: LabelValues.`3`[Counter[F]],
+    topicsLatency: LabelValues.`1`[Summary[F]],
```

Comment:
other metric names have type as suffix, but not this one - why?

Reply:
I did not change the naming scheme, only replaced `Summary` with `Histogram` where applicable